doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -619,32 +619,44 @@ file is a particular case of a module called a *library file*.
 
 .. cmd:: Declare ML Module {+ @string }
 
-   This commands dynamically loads OCaml plugins dynamically.  The string
-   argument must be a valid `findlib <http://projects.camlcity.org/projects/findlib.html>`
-   plugin name, for example ``coq-core.plugins.ltac``. The first component of
-   the plugin name is a package that has to be in scope of ``findlib``.
+   This commands will load an OCaml "plugin" and its dependencies
+   dynamically. The string argument must be a valid `findlib
+   <http://projects.camlcity.org/projects/findlib.html>` library name,
+   for example ``coq-core.plugins.ltac``. The first component of the
+   plugin name is a package that has to be in scope of ``findlib``'s
+   search path.
```

Comment:
Should mention the dune hack here (the part before the colon).

"a valid findlib library name" - "valid" only in syntax or does it have to show up when you run findlib?  If it's the latter,
probably should give the findlib command users should run.  I've never used the command and I don't seem to have it installed, so users likely would benefit from slightly more explanation.  If it's merely syntactic, just state the required syntax.

Reply:
Valid in the sense that indeed, it both syntactically correct, and Coq can find it when the file is compiled.